website/components/plugin-tier-label/style.module.css

```diff
@@ -29,12 +29,6 @@ to match Terraform Registry tier labels.
   }
 }
 
-/* add margin-top to separate from the adjacent
-search bar present on docs pages */
-:global(.g-search) + .root {
-  margin-top: 40px;
-}
-
```

Comment:
These global styles seem to have been the cause of the layout shift around the plugin tier label, so I've removed them. Spacing is handled by a `<br />` element added at the same time as the plugin tier label - at build time.

Reply:
I noticed the layout shift just today and it made me sad. I am thrilled to see that you were already ahead of this! 💪